datenguidepy/output_transformer.py

```diff
@@ -188,11 +194,46 @@ def _determine_column_order(
         ]
         return join_col_list + value_columns + remaining_cols
 
-    def transform(self) -> pd.DataFrame:
+    @staticmethod
+    def _make_verbose_statistic_names(
+        output: pd.DataFrame, meta: QueryResultsMeta
+    ) -> pd.DataFrame:
+        descriptions = cast(StatMeta, meta["statistics"])
+        name_changes = {
+            statistic: f"{descriptions[statistic]} ({statistic})"
+            for statistic in descriptions
+        }
+        return output.rename(columns=name_changes)
+
+    @staticmethod
+    def _make_verbose_enum_values(
+        output: pd.DataFrame, meta: QueryResultsMeta
+    ) -> pd.DataFrame:
+        enum_mappings = copy.deepcopy(cast(EnumMeta, meta["enums"]))
+        for enum in enum_mappings:
+            enum_mappings[enum][None] = "Gesamt"
+        return output.assign(
+            **{
+                col: lambda df: df[col].map(description_map)
+                for col, description_map in enum_mappings.items()
+            }
+        )
+
+    def transform(
+        self, verbose_statistic_names=False, verbose_enum_values=False
+    ) -> pd.DataFrame:
         """Transform the queries results into a Pandas DataFrame.
 
         Returns:
             DataFrame -- Returns a pandas DataFrame of the queries results.
         """
         output = self._convert_results_to_frame(self.query_response)
+        if verbose_statistic_names:
```

Comment:
super useful, these verbose functions!

I was just wondering, if it would be better to change the name of the column STATISTIC_Value... in fact, is there a need to have a column with the JSON response anyway (not really concerning this pull request)?
![grafik](https://user-images.githubusercontent.com/18367515/68072839-6ac32980-fd8a-11e9-8b17-bbc86aa02d7e.png)

Reply:
Good catch this is actually a bug I introduced when refactoring the output transformer. I committed a fix for it.